### PR TITLE
[HOPSWORKS-1617] Azure resolve hostname fix

### PR DESCRIPTION
**Pull request by jimdowling**
https://logicalclocks.atlassian.net/browse/HOPSWORKS-1617

### CODE REVIEW

File: templates/default/core-site.xml.erb [hunk 1]
```diff
@@ -360,4 +360,22 @@
   </property>
 <% end -%>
 
+<% if !node['install']['cloud'].eql? "azure" -%>
```
Comment (SirOibaf) [inline templates/default/core-site.xml.erb]:
```suggestion
<% if node['install']['cloud'].eql? "azure" -%>
```

Comment (SirOibaf) [reply templates/default/core-site.xml.erb]: I guess that's what you meant

File: Karamelfile [hunk 2]
```diff
@@ -8,17 +8,16 @@ dependencies:
       - ndb::mysqld
   - recipe: hops::nn
     global:
+      - hops::rm
```
Comment (SirOibaf) [inline Karamelfile]: We already discussed about this.

File: Karamelfile [hunk 3]
```diff
@@ -8,17 +8,16 @@ dependencies:
       - ndb::mysqld
   - recipe: hops::nn
     global:
+      - hops::rm     
       - ndb::mysqld_tls
       - kagent::default
-      - hops::ndb
```
Comment (SirOibaf) [inline Karamelfile]: This needs to be restored, otherwise the you'll start the namenode without the schema

File: recipes/nm.rb [hunk 4]
```diff
@@ -29,13 +29,13 @@
 
 
 
-if node['install']['cloud'].casecmp?("gce") == 0
+if node['install']['cloud'].casecmp?("gcp") == 0
```
Comment (SirOibaf) [inline recipes/nm.rb]: https://github.com/logicalclocks/conda-chef/blob/a136886cb127b68b2d1e2530a900a504b0a5b5fe/metadata.rb#L93 Can you fix also this line, so users know what to set?

Comment (jimdowling) [reply recipes/nm.rb]: ok, had done that.

File: templates/default/core-site.xml.erb (hunk 1 above)
Comment (jimdowling) [reply templates/default/core-site.xml.erb]: thanks

**Pull request merged**